src/mame/trs/coco12.cpp

```diff
@@ -654,6 +812,12 @@ ROM_START(coco2)
 	ROM_LOAD("extbas11.rom",    0x0000, 0x2000, CRC(a82a6254) SHA1(ad927fb4f30746d820cb8b860ebb585e7f095dea))
 ROM_END
 
+ROM_START(deluxecoco)
+	ROM_REGION(0x8000,MAINCPU_TAG,0)
+	ROM_LOAD("bas20.rom",   0x2000, 0x2000, NO_DUMP CRC(d8f4d15e) SHA1(28b92bebe35fa4f026a084416d6ea3b1552b63d3))
+	ROM_LOAD("extbas20.rom",    0x0000, 0x2000, NO_DUMP CRC(a82a6254) SHA1(ad927fb4f30746d820cb8b860ebb585e7f095dea))
+ROM_END
```

Comment:
Is it even valid to have `NO_DUMP` and a CRC/SHA-1?  Usually `NO_DUMP` has no expected digest at all.

Reply:
Good to know. Fixed.